std/assembly/util/math.ts

```diff
@@ -0,0 +1,1919 @@
+/** Lookup data for exp2f **/
+
+// @ts-ignore: decorator
+@lazy const EXP2F_TABLE_BITS = 5;
+// @ts-ignore: decorator
+@lazy const exp2f_data_tab: u64[] = [
+  /**
+   * exp2f_data_tab[i] = uint(2^(i/N)) - (i << 52-BITS)
+   * used for computing 2^(k/N) for an int |k| < 150 N as
+   * double(tab[k%N] + (k << 52-BITS))
+   */
+  0x3FF0000000000000, 0x3FEFD9B0D3158574, 0x3FEFB5586CF9890F, 0x3FEF9301D0125B51,
+  0x3FEF72B83C7D517B, 0x3FEF54873168B9AA, 0x3FEF387A6E756238, 0x3FEF1E9DF51FDEE1,
+  0x3FEF06FE0A31B715, 0x3FEEF1A7373AA9CB, 0x3FEEDEA64C123422, 0x3FEECE086061892D,
+  0x3FEEBFDAD5362A27, 0x3FEEB42B569D4F82, 0x3FEEAB07DD485429, 0x3FEEA47EB03A5585,
+  0x3FEEA09E667F3BCD, 0x3FEE9F75E8EC5F74, 0x3FEEA11473EB0187, 0x3FEEA589994CCE13,
+  0x3FEEACE5422AA0DB, 0x3FEEB737B0CDC5E5, 0x3FEEC49182A3F090, 0x3FEED503B23E255D,
+  0x3FEEE89F995AD3AD, 0x3FEEFF76F2FB5E47, 0x3FEF199BDD85529C, 0x3FEF3720DCEF9069,
+  0x3FEF5818DCFBA487, 0x3FEF7C97337B9B5F, 0x3FEFA4AFA2A490DA, 0x3FEFD0765B6E4540
+];
+
+/**
+ * ULP error: 0.502 (nearest rounding.)
```

Comment:
Similarly, this doesn't make much sense as documentation, but rather looks like an implementation detail comment.